test/integration/policy_comp_operator_test.go

```diff
@@ -132,7 +132,7 @@ func complianceScanTest(scanPolicyName string, scanPolicyURL string, scanName st
 				)
 
 				return compliancesuite.Object["status"].(map[string]interface{})["phase"]
-			}, common.MaxTimeoutSeconds, 1).Should(Equal("AGGREGATING"))
+			}, common.MaxTimeoutSeconds, 1).Should(Or(Equal("AGGREGATING"), Equal("RUNNING")))
 		})
 		It("ComplianceSuite "+scanName+" scan results should be DONE", func() {
```

Comment:
I think we also want to remove this `It`.

Reply:
you mean.. 
```
		It("ComplianceSuite "+scanName+" scan results should be AGGREGATING", func() {
			By("Checking if ComplianceSuite " + scanName + " scan status.phase is AGGREGATING")
			Eventually(func() interface{} {
				compliancesuite := utils.GetWithTimeout(
					clientManagedDynamic,
					common.GvrComplianceSuite,
					scanName,
					"openshift-compliance",
					true,
					defaultTimeoutSeconds,
				)

				return compliancesuite.Object["status"].(map[string]interface{})["phase"]
			}, common.MaxTimeoutSeconds, 1).Should(Or(Equal("AGGREGATING"), Equal("RUNNING")))
		})
``` ?

Reply:
@mprahl is referring to the next `It()` checking for `DONE`, since if it's still running, it also won't be done for a while and we should just move on.

Reply:
oh after this is clean up so `DONE` is meaningless ok!! Let me change